05-extra.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: page
+title: Version Control with Git
+subtitle: Additional Topics
```

Comment:
Ah, I see. I'll take it on board to use that tool in future.

If anyone has suggestions for the minutes for `{01..04}-*.md`, then I'll add them in later and submit a separate pull request just for that. In the meantime, I don't like the inconsistency of adding the minutes to just one single file which isn't even core material.